kakao/src/main/kotlin/com/agoda/kakao/Assertions.kt

```diff
@@ -5,6 +5,8 @@ import android.graphics.drawable.Drawable
 import android.support.annotation.ColorRes
 import android.support.annotation.DrawableRes
 import android.support.annotation.StringRes
+import android.support.design.widget.BottomNavigationView
+import android.support.test.espresso.NoMatchingViewException
```

Comment:
I didn't see the usage of this import

Reply:
Probably it got imported when I was auto-implementing anonymous object of class `ViewAssertion`, then converted to lambda and forgot to remove it.